### DISCUSSION TITLE
blog/2012/07/02/gutenberg-ipsum/index

### DISCUSSION THREAD

**Discussion by giscus** [andrewheiss/ath-quarto#62] [Blog comments]
# blog/2012/07/02/gutenberg-ipsum/index

Stop using boring, boilerplate Lorem ipsum filler text and build your own random, semi-coherent text from Project Gutenberg books.

https://www.andrewheiss.com/blog/2012/07/02/gutenberg-ipsum/index.html

**Comment by markfullmer** [comment]
There's also a website for this, generating placeholder text from Project Gutenberg, useful when English text is needed for testing natural language processing/search: https://lorem-gutenberg.markfullmer.com/
